### PR TITLE
Add pipeline fixes for ODLC state

### PR DESCRIPTION
**Pull request by Ooglely**
# Add pipeline fixes for ODLC state

FIxes the camera.json output to be usable for vision pipeline function and adds a vision unit test for checking photos

### CODE REVIEW

File: vision/unit_tests/photo_model_test.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            print("Loading image")
+            image: consts.Image = cv2.imread(image_path)
+
+            # Get the camera parameters from the loaded parameter file
+            print("Loading camera parameters")
+            camera_parameters: consts.CameraParameters = image_parameters[image_path]
+
+            # Append all discovered standard objects to the list of saved odlcs
+            print("Finding standard objects")
+            saved_odlcs += std_obj.find_standard_objects(image, camera_parameters, image_path)
```
Comment (cjhr95) [inline vision/unit_tests/photo_model_test.py:71]: For some types in Python, this operator isn't the exact same as append(), so make sure its adding the standard objects to this list in the way you want.

File: vision/unit_tests/photo_model_test.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+            # Append all discovered humanoids to the list of saved humanoids
+            print("Finding emergent objects")
+            saved_humanoids += emg_obj.find_humanoids(
```
Comment (cjhr95) [inline vision/unit_tests/photo_model_test.py:75]: Same thing here, double check += vs. append()

File: vision/unit_tests/photo_model_test.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        for humanoid in saved_humanoids:
+            i += 1
+            emergent_image: consts.Image = cv2.imread(humanoid.get_attribute("image_path"))
+            emergent_crop: consts.Image = crop_image(emergent_image, humanoid)
+
+            cv2.imwrite(f"emergent_object{i}.jpg", emergent_crop)
+
+
+logging.basicConfig(filename="/dev/stdout", level=logging.INFO)
+flyover_pipeline("/home/alen/Documents/temp/photos/jun8/camera.json")
```
Comment (cjhr95) [inline vision/unit_tests/photo_model_test.py]: This path will need to be changed for the actual Multirotor ground station, or you can generalize it using functions from the path module

**Review by cjhr95 — DISMISSED** [review]

File: vision/unit_tests/photo_model_test.py (hunk 1 above)
Comment (Ooglely) [reply vision/unit_tests/photo_model_test.py:71]: This is copied directly from the flyover_vision_pipeline file, and I tested and made sure it worked. I would rather keep it the same as the pipeline file just for consistency

File: vision/unit_tests/photo_model_test.py (hunk 3 above)
Comment (Ooglely) [reply vision/unit_tests/photo_model_test.py]: Changing it to the standard path for camera.json

**Review by cjhr95 — APPROVED** [review]

**Pull request merged**